config/components/namespace/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
```

Comment:
I really like the changes inside of the `config` directory. If we make similar change in k8ssandra-operator, will we then be able to install both operators configured to be cluster-scoped and deployed into their own, respective namespaces?

Reply:
With a single kustomize file or in general? I think creating a layer on top would sadly overwrite the namespace for both, so it would require two kustomize builds..